src/ServiceControl/MessageFailures/Api/FailedMessageViewIndex.cs

```diff
@@ -30,6 +31,7 @@ select new
                     message.Status,
                     TimeSent = (DateTime) processingAttemptsLast.MessageMetadata["TimeSent"],
                     ReceivingEndpointName = ((EndpointDetails) processingAttemptsLast.MessageMetadata["ReceivingEndpoint"]).Name,
+                    QueueAddress = processingAttemptsLast.FailureDetails.AddressOfFailingEndpoint.ToLowerInvariant(),
```

Comment:
@boblangley why are we `ToLowerInvariant()` in an index ?
We should do that as part of the display not the index